hr_attendance_geolocation/models/hr_attendance.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class HrAttendance(models.Model):
+    _inherit = "hr.attendance"
+
+    check_in_latitude = fields.Float(digits="Location", readonly=True)
+    check_in_latitude_raw = fields.Char(
+        "Check-in Latitude", compute="_compute_check_in_latitude_raw"
+    )
+    check_in_longitude = fields.Float(digits="Location", readonly=True)
+    check_in_longitude_raw = fields.Char(
```

Comment:
This is the opposite of raw. Please call it `check_in_longitude_human` or `check_in_longitude_text` or `check_in_longitude_readable`.